shoop_workbench/settings/base_settings.py

```diff
@@ -58,7 +59,6 @@
     'filer',
     'registration',
     'rest_framework',
-
     'shoop.discount_pricing'
```

Comment:
Should we just remove shoop.discount_princg from installed apps since it's not recommended to use it anymore.